index.html

```diff
@@ -19156,25 +19168,25 @@ <h2 id="AudioWorklet-Sequence">
             <a>AudioWorkletProcessor</a>.
           </p>
           <ol>
-            <li>In the main scope, <code>window.audioWorklet</code> is
-            requested to import a script. No <a>AudioWorkletGlobalScope</a>s
-            exist yet, so the script is fetched and added to the Worklet module
-            responses map.
-            </li>
             <li>An <a>AudioContext</a> is created.
             </li>
-            <li>An <a>AudioWorkletGlobalScope</a> is created in association
-            with the context's audio rendering thread. This is the global scope
-            in which <a>AudioWorkletProcessor</a> class definitions will be
-            evaluated.
+            <li>In the main scope, <code>context.audioWorklet</code> is
+            requested to add a script module.
+            </li>
+            <li>Since none exists yet, a new <a>AudioWorkletGlobalScope</a> is
```

Comment:
Although not normative text, we should explain what happens on the second call, when the global scope exists.

Reply:
+1